src/components/Chat/MessageList/index.js

```diff
@@ -6,21 +6,46 @@ import Message from './Message'
 const MessageList = props => {
   const { posts, currentUserId, getUserNamebyId, getIconColor } = props
 
+  let previousDate = 'not_defined'
```

Comment:
Is 'not_defined' used somewhere as a string? Maybe easier to check if previous date is defined or not, if initial value is for example `null`?